routing/control_tower.go

```diff
@@ -1,7 +1,11 @@
 package routing
 
 import (
+	"sync"
+
+	"github.com/go-errors/errors"
```

Comment:
regular "errors" package should suffice?

Reply:
fixed